### PR TITLE
fix(DB/Quest): gameobject loot and related responsible spell script

### PR DESCRIPTION
**Pull request by Helias**
Thanks @r0m1ntik94 for the fix.

## ISSUES ADDRESSED:
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/3224

## TESTS PERFORMED:
- executed successfully the SQL query into the database

## HOW TO TEST THE CHANGES:
- complete the quest https://www.wowhead.com/quest=12713/betrayal

## Target branch(es):
- [x] Master

 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1598302868929248922.sql [hunk 1]
```diff
@@ -0,0 +1,4 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1598302868929248922');
+UPDATE `gameobject_loot_template` SET `QuestRequired` = 0 WHERE `item` = 43059;
+INSERT INTO `spell_target_position` VALUES
+(57553, 0, 571, 5875.43, -1981.37, 234.671, 0, 0); -- Escape Voltarus
```
Comment (FrancescoBorzi) [inline data/sql/updates/pending_db_world/rev_1598302868929248922.sql:4]: missing `DELETE` before `INSERT`

**Review by FrancescoBorzi — CHANGES_REQUESTED** [review]

File: data/sql/updates/pending_db_world/rev_1598302868929248922.sql (hunk 1 above)
Comment (Helias) [reply data/sql/updates/pending_db_world/rev_1598302868929248922.sql]: done

**Review by FrancescoBorzi — APPROVED** [review]

**Comment by felwyn** [comment]
Have to complete the quest chain because it requires special abilities for my character before I encounter Drakuru

Hang on guys!

**Comment by felwyn** [comment]
Completed the quest. Got Drakuru's Skull and Spell was working too.

**Pull request merged**